pages/index.md

```diff
@@ -4,3 +4,6 @@ slug: index
 type: text
 template: homepage.tmpl
 ---
+<div class="homepage-blog-band">
```

Comment:
I don't think we need this `div` here. We can move it to to homepage-blog-band.tmpl and wrap the block with it like this:

```
<div class="blog-box">
      {% block content %}
        {{ post.text() }}
      {% endblock %}
    </div>
 ```

Reply:
I *told* you I was bad at HTML/CSS. Implemented, looks great, much simpler, thanks!